libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -45,7 +41,7 @@ internal class RouteRefreshController(
      */
     fun restart() {
         stop()
-        if (routeRefreshOptions.enabled) {
+        if (tripSession.route?.routeOptions()?.enableRefresh() == true) {
```

Comment:
we may add extension
`fun DirectionsRoute?.enableRefresh() : Boolean`

Reply:
I don't think that's worth it, the same argument could be made for all other `RouteOptions`.